modeling_tool.cc

```diff
@@ -34,40 +34,62 @@ void ReadWriteTransaction(google::cloud::spanner::Client client) {
   namespace spanner = ::google::cloud::spanner;
```

Comment:
Could you move them out of method? You might want to put all "using" statement into an anonymous namespace out of the function.